src/winetricks

```diff
@@ -718,6 +718,26 @@ w_expand_env()
     winetricks_early_wine cmd.exe /c echo "%$1%"
 }
 
+# Check if $1 is executable (output custom message using w_debug if $2 is specified) and return 126 (standart non-executable exit code) or 0 if executable
+w_check_exec()
+{
+    w_executable="$1"
+    w_message="$2"
+
+    if ! command -v "$w_executable" 2>/dev/null; then
+        if [ -n "$w_message" ]; then
+            printf '%s\n' "$w_message"
```

Comment:
This should be w_warn

Reply:
It's expected to use this for logic for example if p7zip (linux) is not executable -> Use windows 7-Zip

So it's made to return `0` or `126` for logic.  -> I don't think that we should bother user with `w_warn()` assuming GUI used.